### DISCUSSION TITLE
React vs Ang

### DISCUSSION THREAD

**Discussion by vipulkumarstm** [community/community#67298] [Discussions]
### Select Topic Area

Question

### Body

Define

**Comment by SURYAROUSHAN** [comment]
Read this
https://radixweb.com/blog/react-vs-angular

**Comment by Umut-Polat-GM** [comment]
Watch this
https://www.youtube.com/watch?v=CynLOUL9vsU&ab_channel=KodapsAcademy

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
